### PR TITLE
Patch fr translation 

### PR DESCRIPTION
**Pull request by Potjoe-97**

### CODE REVIEW

File: src/public/translations/fr/translation.json [hunk 1]
```diff
@@ -111,7 +112,7 @@
     "collapseSubTree": "réduire le sous-arbre",
     "tabShortcuts": "Raccourcis des onglets",
     "newTabNoteLink": "(ou clic central de la souris) sur le lien de la note ouvre la note dans un nouvel onglet",
-    "onlyInDesktop": "Uniquement sur ordinateur (version Electron)",
+    "onlyInDesktop": "Only in desktop (Electron build)",
```
Comment (eliandoran) [inline src/public/translations/fr/translation.json:115]: @Potjoe-97 , shouldn't this be in French? I see the English translation was carried over instead.

File: src/public/translations/fr/translation.json [hunk 2]
```diff
@@ -165,7 +166,8 @@
     "textImportedAsText": "Importez HTML, Markdown et TXT sous forme de notes de texte si les métadonnées ne sont pas claires",
     "codeImportedAsCode": "Importez des fichiers de code reconnus (par exemple <code>.json</code>) en tant que notes de code si cela n'est pas clair à partir des métadonnées",
     "replaceUnderscoresWithSpaces": "Remplacez les tirets bas par des espaces dans les noms de notes importées",
-    "import": "Importer"
+    "import": "Importer",
+    "failed": "Import failed: {{message}}."
```
Comment (eliandoran) [inline src/public/translations/fr/translation.json:170]: Same here, not translated.

File: src/public/translations/fr/translation.json [hunk 3]
```diff
@@ -620,18 +622,21 @@
...
-    "export_note": "Exporter la note",
+    "export_note": "Exporter",
     "delete_note": "Supprimer la note",
-    "print_note": "Imprimer la note",
-    "save_revision": "Enregistrer la version"
+    "print_note": "Imprimer",
+    "save_revision": "Enregistrer une version",
+    "convert_into_attachment_failed": "Converting note '{{title}}' failed.",
+    "convert_into_attachment_successful": "Note '{{title}' has been converted to attachment.",
+    "convert_into_attachment_prompt": "Are you sure you want to convert note '{{title}}' into an attachment of the parent note?"
```
Comment (eliandoran) [inline src/public/translations/fr/translation.json:639]: Need translation.

File: src/public/translations/fr/translation.json [hunk 4]
```diff
@@ -620,18 +622,21 @@
   },
   "note_actions": {
     "convert_into_attachment": "Convertir en pièce jointe",
-    "re_render_note": "Re-rendre la note",
+    "re_render_note": "Recharger la note",
     "search_in_note": "Rechercher dans la note",
-    "note_source": "Source de la note",
+    "note_source": "Code source",
```
Comment (eliandoran) [inline src/public/translations/fr/translation.json:627]: Need translation.

File: src/public/translations/fr/translation.json [hunk 5]
```diff
@@ -1110,7 +1125,7 @@
     "custom_name_label": "Nom du moteur de recherche personnalisé",
     "custom_name_placeholder": "Personnaliser le nom du moteur de recherche",
     "custom_url_label": "L'URL du moteur de recherche personnalisé doit inclure {keyword} comme espace réservé pour le terme de recherche.",
-    "custom_url_placeholder": "Personnaliser l'URL du moteur de recherche",
+    "custom_url_placeholder": "Customize search engine url",
```
Comment (eliandoran) [inline src/public/translations/fr/translation.json:1128]: Need translation.

File: src/public/translations/fr/translation.json [hunk 6]
```diff
@@ -1289,6 +1304,8 @@
     "insert-child-note": "Insérer une note enfant",
     "delete": "Supprimer",
     "search-in-subtree": "Rechercher dans le sous-arbre",
+    "hoist-note": "Hoist note",
+    "unhoist-note": "Unhoist note",
```
Comment (eliandoran) [inline src/public/translations/fr/translation.json:1308]: Need translation.

**Review by eliandoran — CHANGES_REQUESTED** [review]

**Comment by hasecilu** [comment]
I don't know if you used the [translation](https://github.com/TriliumNext/Notes/blob/develop/bin/translation.sh) script but when using it, more specifically the `--update2` flag you need to make sure all strings are translated because otherwise PO editors could use the English string is there is no translation and there is no checks to avoid repeating source strings.

**Comment by Potjoe-97** [comment]
Sorry for the draft state of the PR. I use Crowdin to help me in the translation, and I updated the original source files before exporting my fixes for the last translation, resulting in numerous untranslated strings. Moreover, I'm a bit new to git, and I do have some difficulties managing branches / pr... Still learning :) @eliandoran I'm updating the files rn, all strings should be translated.

**Review by eliandoran — APPROVED** [review]
Thank you for your work!

**Pull request merged**